apps/src/javalab/JavalabView.jsx

```diff
@@ -402,9 +404,11 @@ class JavalabView extends React.Component {
                     isEditingStartSources={isEditingStartSources}
                     disableFinishButton={disableFinishButton}
                     disableRunButtons={awaitingContainedResponse}
-                    onContinue={onContinue}
+                    onContinue={() => onContinue(this.props.isSubmittable)}
```

Comment:
nit: pull `this.props.isSubmittable` out of props on line 297 so this becomes `onContinue(isSubmittable)`